### DISCUSSION TITLE
Command line interface how to name models of a pipeline

### DISCUSSION THREAD

**Discussion by smisra1** [explosion/spaCy#10790] [Help: Coding & Implementations] [answered]
I wanted to train 2 spancat models in 1 pipeline.
The current cli command i use to create 1 spancat pipe is as the following:
```
subprocess.run([sys.executable,
                "-m", "spacy", "init",
                "config", "output_2/configs.cfg",
                "--lang", "en",
                "--pipeline", "spancat",
                "--optimize", "efficiency"])
```

I wanted to add another spancat model and name them to differentiate them: 'spancat_1' and 'spancat_2'. I could'nt find the command to name them. How could I add the name for them?
Thanks.

**Comment by adrianeboyd** [comment, marked as answer]
This isn't currently supported, so you'd need to edit your config by hand to add the second `spancat` component.

Our to-do list includes adding an `init config` option to support this, which we were thinking would look something like this:

`--pipeline tagger,spancat,spancat --name tagger,spancat1,spancat2`

**Reply by smisra1** [reply]
Noted. Looking forward to it!
